### PR TITLE
fix: footer attributes

### PR DESCRIPTION
**Pull request by breadadams**

### CODE REVIEW

File: src/components/patterns/Footer/Footer.js [hunk 1]
```diff
@@ -141,15 +141,15 @@ const Footer = ({ isDark, ...props }) => {
             {[
               { children: 'API', href: '/docs/api/getting-started/overview' },
               { children: 'Blog', href: '/blog' },
-              { children: 'Chat', href: '/community' },
+              { children: 'Community', href: '/community' },
```
Comment (breadadams) [inline src/components/patterns/Footer/Footer.js:144]: @Kikobeats was this rename intentional?

Comment (Kikobeats) [reply src/components/patterns/Footer/Footer.js:144]: Yes, the chat is now Slack 🙂
